classes/template/PKPTemplateManager.php

```diff
@@ -1261,6 +1267,37 @@ public function fetchJson($template, $status = true)
      */
     public function display($template = null, $cache_id = null, $compile_id = null, $parent = null)
     {
+
+        if($this->isBackendPage) {
+            $this->unregisterPlugin('modifier', 'escape');
+
+            /** prevent {{ JS }} injection  */
+            $this->registerPlugin('modifier', 'escape', function ($string, $esc_type = 'html', $char_set = 'ISO-8859-1') {
+                $result = $string;
+                if($esc_type === 'html') {
+                    $result = $this->smartyEscape($result, $esc_type, $char_set);
+                    $result = str_replace('{{', '<span v-pre>{{</span>', $result);
+                    $result = str_replace('}}', '<span v-pre>}}</span>', $result);
```

Comment:
It's possible to unify if you want: `str_replace(['{{', '}}'], ['<span v-pre>{{</span>', '<span v-pre>}}</span>'], $result);`